### ISSUE TITLE
Tab gets wider on hover (when there are many tabs) - inline_tab_audio_icons.css

### ISSUE THREAD

**Issue by corobin** [MrOtherGuy/firefox-csshacks#138]
Thanks for the collection of CSS hacks, they are very helpful to fix Mozilla's bad UI.

I noticed a small edge-case when using `inline_tab_audio_icons.css` (unmodified, no other styles) on Fx91 (default new profile, on Win10). When tabs are at minimum width, and there is the audio icon on the active tab, hovering over the active tab makes it jump wider.

Steps to reproduce:

1. apply inline_tab_audio_icons.css style
2. open many tabs - so that the tabs are at minimum width
3. select any tab and open a youtube video so that the current tab shows the audio icon
4. hover over the current tab

result: upon hovering over it, the tab jumps to be wider

expected result: the tab should not jump to be wider

screenshots:

mouse is not hovering over current active tab
https://i.imgur.com/UjNECsF.png

mouse is hovering over current active tab
https://i.imgur.com/EkjawK2.png

It looks like there is extra padding that is added between the audio icon and the X button on the tab when mouse is hovering over the tab, which pushes the tab wider.

this creates a bit of an annoying effect of the tab "growing" when hovering over it, but not consistently since it does not happen unless the tab is small enough first.

is this possible to be fixed or maybe is there a workaround?

**Comment by MrOtherGuy** [comment]
Thanks for reporting. Yeah, the default tab min width (76px) is not wide enough to fit favicon, audio icon and close button. Tab min width is controlled (to some extent with pref `browser.tabs.tabMinWidth`)

I made a few changes in e2a2974 that makes the icon bigger, to increase it's visibility which is more important now if you don't want the secondary label to show up. Simultaneously, I made it pack more tightly against favicon and the tab label, this in turn makes it fit just barely enough to 76px to not cause the tab to overflow when hovered. Of course, you would still face the issue if you were to make tab min width smaller than 76px - but in that case one probably shouldn't be using this style anyway since because it wastes space.
